src/components/Map.js

```diff
@@ -33,10 +33,7 @@ export const Map = ({
     const lat = event?.center?.lat();
     const lng = event?.center?.lng();
     const newMapCenter = { lat, lng };
-
-    window.setTimeout(() => {
-      setNewCenterMap(newMapCenter);
-    }, 1000);
+    setNewCenterMap(newMapCenter);
```

Comment:
This feels better as an end-user, for sure! Do we have another mechanism for making sure we don't make multiple API calls at one time?

Reply:
Yes, there are two mechanisms (I think):

1. Our LocationsContext, we make sure that one call is executed in all our app, and share the results.
2. Using 'AbortController' in our custom hook `useLocations`, if we're calling multiple times, they're canceled 'automatically'.

**Canceled requests**

![image](https://user-images.githubusercontent.com/8689897/128267344-c103907b-d77e-4f11-b598-c8bdf4c164e6.png)

Reply:
This is nice and clean. I like it. And I was able to make it work to cancel an API call twice and watch based on this method. Thank you for asking, @segdeha and thank you for the screen shot, @alxmcr.